docs/modeling/rebac-GHC.mdx

```diff
@@ -1,19 +1,253 @@
+
+- **[RBAC](https://docs.permit.io/category/rbac)**
+- **[ABAC](https://docs.permit.io/category/abac)**
+- **[ReBAC](https://docs.permit.io/category/rebac)**
+
+:::note
+A highly detailed explanation and walkthrough of this application is available in [this blog](https://www.permit.io/blog/building-healthcare-authorization-nextjs)
+The Git repository for this demo application [is available here](https://github.com/permitio/Galactic-Health-Corporation).
+:::
+
+### GHC App Key Features:
+
+- **User Roles**: GHC accommodates various user types, each assigned with a corresponding role. These include doctors, patients, caretakers, and administrators.
+  Each user type has distinct responsibilities and access levels tailored to their role.
+
+- **Patient Management**: Patients can access their own health records, medical history, and treatment plans. They can also grant limited access to caregivers or family members as needed.
+
+- **Doctor-Patient Relationships**: Doctors can view and manage the medical records of patients under their care. Patient data is only accessible to authorized medical professionals.
+
+- **Attribute-Based Access**: Access to certain resources within the application is restricted by using attributes like geographic location, patient age, date scoping, or treatment status -
+  making sure they are only accessible under the right conditions.
+
+- **Feature testing pilot Group**: GHC allows the management of user feature testing pilot groups, which allows them to test new application features based on users fitting specific criteria.
+
+- **Delegated Permissions**: Users can delegate limited access to their health information to others, such as caregivers, with specific limitations.
+
+## Getting Started
+
+### Prerequisites
+
+1. Node.js Environment -
+   You'll need a working [Node.js](https://nodejs.org/en/download) environment installed on your local machine.
+
+2. Docker Environment -
+   GHC relies on Docker for running the decision point container locally. Make sure you have [Docker installed and configured](https://docs.docker.com/engine/install/).
+
+3. Clerk Account -
+   GHC uses [Clerk](http://clerk.com/) to authenticate users, allowing them to log into the app.
+
+4. Permit.io Account -
+   GHC leverages Permit.io for managing user permissions. You'll need a [Permit.io](http://app.permit.io/) account to configure and manage access control.
+
+### Setting up the application
+
+1. Clone the repository to your local machine.
+
+2. In the root folder, create a file named `.env.local` and copy the content of `.env.example` to it.
+
+3. From the Clerk dashboard, go to `API Keys`, choose Next.js example, and copy the `NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY` and `CLERK_SECRET_KEY` values to the `.env.local` file.
+
+4. In the Permit dashboard, go to `Connect`, and copy the token variable to the `PERMIT_SDK_KEY` value into the `.env.local` file.
+
+5. Run `npm install` to install the application dependencies.
```

Comment:
For example this should say:

Install the required dependancies.

(and then the code snippet for `npm install`